adaptors/proximityadaptor-evdev/proximityadaptor-evdev.cpp

```diff
@@ -38,6 +38,7 @@
 
 #define SELFDEF_EV_SW 5
 #define SELFDEF_EV_FRONT_PROXIMITY 11
+#define SELFDEF_EV_DISTANCE 25
```

Comment:
The biggest issue I have with these is that the event codes do not follow the naming convention from input.h and thus it is not obvious at place of use that SELFDEF_EV_DISTANCE is an event code and goes together with EV_ABS type.

 If the fact that it is self defined needs to be highlighted, then at least I would prefer to see

 * SELFDEF_EV_FRONT_PROXIMITY -> SELFDEF_SW_FRONT_PROXIMITY
 * SELFDEF_EV_DISTANCE -> SELFDEF_ABS_DISTANCE

so that the only difference to "standard" defs would be the SELFDEF prefix